gamepanel/game.php

```diff
@@ -560,17 +576,25 @@ function joinBar()
 	function openBar()
 	{
 		global $User;
+		$playBeta = '';
+		if (($User->optInFeatures & 0x1) > 0) { $playBeta = '<a href="board.php?gameID='.$this->id.'/beta #gamePanel" style="margin-left: 40px">'.l_t('Play Beta').'</a> '; }

```

Comment:
i see that `$User->optInFeatures & 0x1) > 0` is being used in two different functions. can we dry this comparison up by assigning it to a variable?